task/net/Cargo.toml

```diff
@@ -10,12 +10,12 @@ cortex-m = {version = "0.7", features = ["inline-asm"]}
 num-traits = {version = "0.2.12", default-features = false}
 serde = {version = "1", default-features = false, features = ["derive"]}
 ssmarshal = {version = "1", default-features = false}
-stm32h7 = {version = "0.13", features = ["stm32h743"]}
+stm32h7 = {version = "0.13"}
 zerocopy = "0.6"
 
 userlib = {path = "../../sys/userlib", features = ["panic-messages"]}
 hubris-num-tasks = {path = "../../sys/num-tasks", features = ["task-enum"]}
-drv-stm32h7-eth = {path = "../../drv/stm32h7-eth", features = ["with-smoltcp", "ipv6", "h743"]}
+drv-stm32h7-eth = {path = "../../drv/stm32h7-eth"}
```

Comment:
This won't work without the ipv6 and with-smoltcp features, so, please move them back up here; only the h743/53 part needs to move below.

Reply:
I understood that the features from the [features] section below replace these. I'll try and see if it's additive instead. I am a strong reader of Rust, but a very new writer, and Cargo is a total mystery to me...